remote/imgui_remote.h

```diff
@@ -116,6 +116,11 @@ struct WebSocketServer : public IWebSocketServer
     }
 	virtual void OnMessage(OpCode opcode, const void *data, int size)
 	{
+		if(ClearInput == 1) {
+			Input.MouseWheelDelta = 0;
+			ClearInput = 0;
```

Comment:
Should this be clearing everything?

Reply:
It should only clear input we don't want to persist between frames. Currently the mouse wheel is the only one (since the websocket doesn't send it over when it hasn't moved)